ui/brave_custom_notification/BUILD.gn

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) 2020 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import("//build/config/features.gni")
+import("//build/config/jumbo.gni")
+import("//build/config/ui.gni")
+import("//testing/test.gni")
+import("//ui/base/ui_features.gni")
+
+source_set("brave_custom_notification") {
+  if (is_win || is_linux || is_mac) {
+    deps = [
+      "//base",
+      "//ui/base",
+      "//ui/strings",
+      "//url",
+    ]
+
+    deps += [
+      "//brave/ui/brave_custom_notification/public/cpp" ,
```

Comment:
Please remove any deps which are no longer needed and add any missing deps that are needed. Use `npm run gn_check` to detect missing deps